src/coreclr/System.Private.CoreLib/src/System/Array.CoreCLR.cs

```diff
@@ -18,6 +17,28 @@ public abstract partial class Array : ICloneable, IList, IStructuralComparable,
         [MethodImpl(MethodImplOptions.InternalCall)]
         private static extern unsafe Array InternalCreate(RuntimeType elementType, int rank, int* pLengths, int* pLowerBounds);
 
+        private static unsafe Array InternalCreateFromArrayType(Type arrayType, int rank, int* pLengths, int* pLowerBounds)
+        {
+            if (rank == 1 && !ContainsLowerBounds(rank, pLowerBounds))
```

Comment:
```suggestion
            if (rank == 1 && pLowerBounds[0] == 0)
```
And delete ContainsLowerBounds